### DISCUSSION TITLE
have the animation tool move column with the arrow keys

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6550] [Ideas]
**Is your feature request related to a problem? Please describe**
I'm always frustrated when i try to move a column with the arrow keys and it goes to another frame

**Describe the solution you'd like**
move column with arrow keys

https://user-images.githubusercontent.com/36406840/133004120-8c1be2e7-8920-4360-a2f7-2d259174d6bc.mp4

this is the solution but with video editing to look like the result desired

**Comment by gab3d** [comment]
I also think it'd be good to have that option for increased precision while placing objects, perhaps by using some modifier key (Shift, Alt, whatever) while te tool is active?

**Comment by RodneyBaker** [comment]
This was implemented via https://github.com/opentoonz/opentoonz/pull/7075
